tests/sqlparser_common.rs

```diff
@@ -2518,7 +2518,7 @@ fn parse_bad_constraint() {
 
 #[test]
 fn parse_scalar_function_in_projection() {
-    let names = vec!["sqrt", "array", "foo"];
```

Comment:
This may be a controversial change for this test. PostgreSQL doesn't have a function-based array constructor (`array(1, 2, 3)`), nor can I find information on other databases using such a syntax. Some, like BigQuery, use the `ARRAY(subquery)` syntax implemented there. It'd be great to hear some thoughts on this.